create_graph.py

```diff
@@ -29,24 +29,40 @@ def create_igraph_graph(edgelist_df,labels):
 
     edgelist_df = edgelist_df[['subject', 'object', 'predicate']]
 
-#    uri_labels = labels[['Identifier']]
-# Different uri column in PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_NodeLabels.txt. 
-    uri_labels = labels[['entity_uri']]
-
     g = Graph.DataFrame(edgelist_df,directed=True,use_vids=False)
 
     g_nodes = g.vs()['name']
 
     return g,g_nodes 
 
 # Wrapper function
-def create_pkl_graph(triples_file,labels_file):
-
-    triples_df,labels = process_pkl_files(triples_file,labels_file)
+# Includes a "kg_type" parameter for graph type. Options include 'pkl' for PheKnowLator and 'kg-covid19' for KG-Covid19
+def create_graph(triples_file,labels_file, kg_type = "pkl"):
+    if kg_type == "pkl":
+        triples_df,labels = process_pkl_files(triples_file,labels_file)
+    elif kg_type == "kg-covid19":
+        triples_df,labels = process_kg_covid19_files(triples_file,labels_file)
+    else:
+        raise Exception('Invalid graph type! Please set kg_type to "pkl" or "kg-covid19"')
 
     g_igraph,g_nodes_igraph = create_igraph_graph(triples_df,labels)
 
     # Created a PKL class instance
     pkl_graph = KnowledgeGraph(triples_df,labels,g_igraph,g_nodes_igraph)
 
     return pkl_graph
+
+
+
+
+
+###Read in all kg_covid19 files, outputs triples and labels as a df
+def process_kg_covid19_files(triples_file,labels_file):
```

Comment:
In the future, may be nice for use to programmatically figure out the filetype so that we don't have many different functions for each KG